src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/RedirectHandler.cs

```diff
@@ -116,7 +127,7 @@ private Uri GetUriForRedirect(Uri requestUri, HttpResponseMessage response)
             {
                 if (NetEventSource.IsEnabled)
                 {
-                    NetEventSource.Error(this, $"Insecure https to http redirect from '{requestUri}' to '{location}' blocked.");
+                    TraceError($"Insecure https to http redirect from '{requestUri}' to '{location}' blocked.", response.RequestMessage.GetHashCode());
```

Comment:
Is this really an error?

Reply:
I think it is likely enough to be a problem for customers that it is worth highlighting.